tests/components/snooz/test_fan.py

```diff
@@ -200,7 +202,14 @@ async def test_restore_state(
     assert state.state == STATE_UNAVAILABLE
 
     # reload entry
-    await create_mock_snooz_config_entry(hass, device)
+    with (
+        patch("homeassistant.components.snooz.SnoozDevice", return_value=device),
+        patch(
+            "homeassistant.components.snooz.async_ble_device_from_address",
+            return_value=generate_ble_device(device.address, device.name),
+        ),
+    ):
```

Comment:
Could this be a Generator fixture (yield) to avoid duplicate code?

Reply:
not really needed.